test/scripts/validation/test.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
The directory this in should probably be named `clusters` since youre testing cluster deployments

Reply:
Mike, you are suggested to call it "validation".

Reply:
Updated.